driver/gl/canvas.go

```diff
@@ -61,7 +61,12 @@ func (c *glCanvas) SetContent(content fyne.CanvasObject) {
 }
 
 func (c *glCanvas) Refresh(obj fyne.CanvasObject) {
-	refreshQueue <- obj
+	select {
```

Comment:
Thanks, a good catch - this fixes the issue for me too (though I think we should look more into why there is no noticeable side effect to simply discarding the overflow!)